### ISSUE TITLE
Counsel projectile (wrong-type-argument arrayp nil) error

### ISSUE THREAD

**Issue by FieryCod** [syl20bnr/spacemacs#11152]
#### Description :octocat:
Counsel projectile stopped working with the newest version of packages.

#### Reproduction guide :beetle:
- Start Emacs
- SPC p f (or any of counsel-projectile command)

*Observed behaviour:* :eyes: :broken_heart:
Every counsel projectile command not work

*Expected behaviour:* :heart: :smile:
Counsel projectile commands should work

#### System Info :computer:
- OS: darwin
- Emacs: 26.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 9af8e1564)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: ivy
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-snippets-in-popup nil auto-completion-return-key-behavior nil auto-completion-enable-help-tooltip t auto-completion-enable-sort-by-usage t auto-completion-tab-key-behavior 'complete spacemacs-default-company-backends
                  '(company-keywords company-files company-dabbrev-code company-dabbrev company-css company-web-html))
 (python :variables python-backend 'anaconda)
 (c-c++ :variables c-c++-default-mode-for-headers 'c++-mode c-c++-enable-clang-support t)
 (git :variables git-magit-status-fullscreen t)
 (version-control :variables version-control-diff-tool 'diff-hl version-control-global-margin t)
 (org :variables org-enable-github-support t)
 (shell :variables shell-default-height 35 shell-default-position 'bottom shell-default-term-shell "/bin/bash")
 colors markdown osx docker php javascript evil-commentary csv better-defaults clojure go html github emacs-lisp syntax-checking neotree yaml parinfer plantuml ruby sql typescript gtags ivy)
```
- System configuration features: JPEG RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS LCMS2

#### Backtrace :paw_prints:
```
Debugger entered--Lisp error: (wrong-type-argument arrayp nil)
  define-key((keymap) nil counsel-projectile-command-map)
  byte-code("\301 \302\001\010\303#\210\302\001\304\305#\210\302\001\306\307#\210\302\001\310\311#\210\302\001\312\313#\210\302\001\314\315#\210\302\001\316\317#\210\211\207" [projectile-keymap-prefix make-sparse-keymap define-key counsel-projectile-command-map [remap projectile-find-file] counsel-projectile-find-file [remap projectile-find-dir] counsel-projectile-find-dir [remap projectile-switch-to-buffer] counsel-projectile-switch-to-buffer [remap projectile-grep] counsel-projectile-grep [remap projectile-ag] counsel-projectile-ag [remap projectile-switch-project] counsel-projectile-switch-project] 5)
  (defvar counsel-projectile-mode-map (byte-code "\301 \302\001\010\303#\210\302\001\304\305#\210\302\001\306\307#\210\302\001\310\311#\210\302\001\312\313#\210\302\001\314\315#\210\302\001\316\317#\210\211\207" [projectile-keymap-prefix make-sparse-keymap define-ke
```

**Comment by FieryCod** [comment]
Ok it's caused by 9c6e9813abec6e067c659e9107bf356086a95e04 commit in `projectile`

**Comment by FieryCod** [comment]
Temporary fix 
```
(setq projectile-keymap-prefix (kbd "C-c C-p"))
```
Please add this to user-config. Projectile nil'ed the keymap prefix which is used by counsel-projectile.

**Comment by bittopaz** [comment]
With the latest develop branch and with all the packages up-to-date, this error is gone.

**Comment by sdwolfz** [comment]
Does that mean #11153 is not needed anymore? @FieryCod can you confirm that please?

**Comment by aaronjensen** [comment]
Confirmed, this doesn't appear to be an issue anymore. This can be closed as can #11153

**Comment by FieryCod** [comment]
@sdwolfz @aaronjensen @bittopaz Issue is gone. Closing this and the #11153. Thank you guys :)
